dynamic-simulation/dynamic-simulation-api/src/main/java/com/powsybl/dynamicsimulation/DynamicSimulationResult.java

```diff
@@ -14,4 +18,13 @@
     boolean isOk();
 
     String getLogs();
+
+    Map<String, TimeSeries> getCurves();
+
+    TimeSeries getCurve(String curve);
+
+    /**
+     * The Timeline contains information about relevant events that may have happened during the time domain simulation.
```

Comment:
I don't understand the purpose of this time series. Is it a double or a string time series. Could you give an example of what is going to contain?

Reply:
It will be a string time series containing a textual description of events that have been considered relevant during the dynamic simulation. An example from Dynawo output (time is given in milliseconds since the start of the simulation):

```
5000	 _BUS____1-BUS____5-1_AC  LINE : opening on side 2
5000	 GEN____3_SM  PMIN : deactivation
5248	 CLA_2_5  CLA : arming by over-current constraint
5405	 GEN____6_SM  PMIN : activation
10248	 CLA_2_5  CLA : order to change topology
10248	 _BUS____2-BUS____5-1_AC  LINE : opening both sides
```

Reply:
Ok thanks it is clear now, so maybe we can use `StringTimeSeries` in the API?

Reply:
Done